flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/BufferManager.java

```diff
@@ -139,10 +141,10 @@ void requestExclusiveBuffers() throws IOException {
 	}
 
 	/**
-	 * Requests floating buffers from the buffer pool based on the given required amount, and returns the actual
+	 * Requests floating buffers from the buffer pool based on the given backlog, and returns the actual
 	 * requested amount. If the required amount is not fully satisfied, it will register as a listener.
 	 */
-	int requestFloatingBuffers(int numRequired) throws IOException {
+	int requestFloatingBuffers(int backlog) throws IOException {
```

Comment:
This renaming is not necessary. 

`BufferManager` is abstracted as a general purpose, not coupled with credit-based process, so it is better to not define a argument strongly related to credit-based purpose. `numRequired` seems more general to describe the semantic.